archlinuxcn/oniux-git/PKGBUILD

```
@@ -0,0 +1,51 @@
+# Maintainer: Mumi Jim <echo "=02bj5yav9Gb0V3bA1Waq9VatVXb" | rev | base64 -d>
+#Thanks Integral and Klaus
+#only for x86_64
+
+pkgname="oniux-git"
+_appname="oniux"
+pkgver=0.4.0
+pkgrel=1
+url='https://gitlab.torproject.org/tpo/core/oniux'
+license=('MIT OR Apache-2.0')
+options=('!lto')
+makedepends=('git' 'cargo')
+arch=('x86_64')
+provides=('oniux')
+conflicts=('oniux')
+source=("git+$url.git")
+sha512sums=('SKIP')
+
+pkgver() {
+	cd "${srcdir}/${_appname}"
+	git describe --long --tags | sed 's/^v//;s/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+prepare() {
+	export RUSTUP_TOOLCHAIN=stable
+	export CARGO_TARGET_DIR=target
+	#Execute 'cargo fetch' to detect Cargo.lock and download and cache dependencies.
+	cargo fetch --locked -target "$(rustc -vV | sed -n 's/host: //p')"
```

```suggestion
	cargo fetch --locked --target "$(rustc -vV | sed -n 's/host: //p')"
```
You should use `--` for long options.